package.json

```
@@ -23,17 +23,14 @@
     "postinstall": "if [ -z \"$CI\" ]; then yarn --cwd=packages/styles && yarn --cwd=packages/react; fi",
     "screenshots": "docker run --rm --ipc=host -v $(pwd)/playwright-ct.config.ts:/app/playwright-ct.config.ts -v $(pwd)/packages:/app/packages -v $(pwd)/e2e:/app/e2e cauldron-playwright-e2e-screenshots",
     "screenshots:docker": "docker build . -t cauldron-playwright-e2e-screenshots -f e2e/Dockerfile --build-arg playwright_version=$(npm list --depth=0 playwright | grep playwright | cut -f2 -d '@')",
-    "release": "./scripts/release.sh"
+    "release": "./scripts/release.sh",
+    "precommit": "lint-staged",
+    "prepare": "husky install"
```

```suggestion
    "prepare": "husky"
```

The install command is deprecated.